chef/cookbooks/neutron/recipes/network_agents_ha.rb

```diff
@@ -125,13 +125,22 @@
 # right CA file as we allow Keystone's and Neutron's to use different CAs.  So
 # we just rely on the correct CA files being installed in a system wide default
 # location.
+file "/etc/neutron/os_password" do
+  owner 'root'
+  group 'root'
+  mode '0600'
+  content keystone_settings["admin_password"]
+  # Our Chef is apparently too old for this :-/
+  #sensitive true
+  action :create
+end
+
 pacemaker_primitive ha_tool_primitive_name do
   agent node[:neutron][:ha][:network][:ha_tool_ra]
   params ({
     "os_auth_url"    => keystone_settings["internal_auth_url"],
     "os_tenant_name" => keystone_settings["admin_tenant"],
     "os_username"    => keystone_settings["admin_user"],
-    "os_password"    => keystone_settings["admin_password"],
```

Comment:
Well actually in the long term it's not necessarily just for the HA tool; it could be for anything which needs to communicate with the API - possibly even stuff outside neutron, e.g. the keystone OCF RA.  So you *could* argue it should be in `/etc/keystone`.  But right now it really shouldn't matter at all.

I can't imagine any reason why it would need to be a configurable parameter, so I suggest we stick with the simplest thing which could possibly work for now.  If we need to make it configurable later then we can easily do that, but that seems like a waste of effort right now.